src/components/compound/ConfirmAddCompoundModalBottom.tsx

```diff
@@ -1,11 +1,10 @@
-import { Fraction, Percent, Token, TokenAmount } from '@ubeswap/sdk'
+import { Token, TokenAmount } from '@ubeswap/sdk'
```

Comment:
The changes to this file and `ConfirmWithdrawCompoundModalBottom.tsx` were done automatically when I ran the linter.